Source: David Legrand <david.legrand@clever-cloud.com>
Upstream: No
Reason: Add failing tests to `skip_on_sandboxed_packaging`

---
 cmd/tools/vtest-self.v | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmd/tools/vtest-self.v b/cmd/tools/vtest-self.v
index 820be2cde..281c3537e 100644
--- a/cmd/tools/vtest-self.v
+++ b/cmd/tools/vtest-self.v
@@ -351,6 +351,8 @@ const skip_on_sandboxed_packaging = [
 	'vlib/v/gen/native/tests/native_test.v',
 	'vlib/v/compiler_errors_test.v',
 	'vlib/v/gen/c/coutput_test.v',
+	'vlib/net/http/server_test.v',
+	'vlib/os/os_test.c.v',
 ]
 
 fn Config.init(vargs []string, targs []string) !Config {
-- 
2.44.0